src/datanode/src/instance.rs

```diff
@@ -207,12 +209,17 @@ pub(crate) async fn new_object_store(store_config: &ObjectStoreConfig) -> Result
         ObjectStoreConfig::Oss { .. } => new_oss_object_store(store_config).await,
     };
 
+    let mem_accessor = MemoryBuilder::default().build().unwrap();
```

Comment:
Looks like we enable caching for all storage types, and use memory as cache. I think it's not necessary for local file storage type. Memory cache may consume too much memory, prefer file as caching for s3/OSS. 

May we can add a new configuration `cache_path: Option<String>` to `S3Config` and `OssConfig`, if it is not `None`, we enable the caching for them.

Reply:
>prefer file as caching for s3/OSS

+1, caching files on local disk can alleviate lots of remote requests (and maybe latency of reading an object) with lower cost.